lib/orbit_web/components/layouts/root.html.heex

```diff
@@ -23,6 +23,9 @@
     <%= if @conn.assigns[:name] do %>
       <meta name="userName" content={@name} />
     <% end %>
+    <%= if full_story_org_id = Application.get_env(:orbit, :full_story_org_id) do %>
```

Comment:
**suggestion / nitpick:** I think it would be a bit more idiomatic, as well as consistent with the other `<meta>` tags, if we put the FullStory org ID in the `conn` assigned rather than directly calling `get_env` here.

Reply:
agreed- f43f3ff